### PR TITLE
Yueying

### PR DESCRIPTION
**Pull request by OoiYueYing**

### CODE REVIEW

File: src/components/DetailCard/DetailCard.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const DetailCard = ({
   price,
   imageLink
 } = {}) => (!title ? (
```
Comment (ooihuiying) [inline src/components/DetailCard/DetailCard.jsx]: you could remove the ! in front of title and swap the positions of the following 2 options.

File: src/config/constant.js [hunk 2]
```diff
@@ -1,25 +1,23 @@
 // Constants
 
 /* LandingFilter options */
-const regionOptions = [
+export const regionOptions = [
   { key: 'North', text: 'North', value: 'North' },
   { key: 'South', text: 'South', value: 'South' },
   { key: 'East', text: 'East', value: 'East' },
```
Comment (ooihuiying) [inline src/config/constant.js:7]: We can add an option for central as well.

File: src/config/constant.js [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
 
-const paxOptions = [
+export const paxOptions = [
   { key: '1-5', text: '1-5', value: '1-5' },
   { key: '6-10', text: '6-10', value: '6-10' },
   { key: '> 11', text: '> 11', value: '> 11' }
 ];
 
-const ratingOptions = [
+export const ratingOptions = [
```
Comment (ooihuiying) [inline src/config/constant.js:17]: Some data does not contain rating. Perhaps we can add **0** as an option.

File: src/components/DetailCard/DetailCard.jsx [hunk 4]
```diff
@@ -29,7 +29,7 @@ const DetailCard = ({
     </div>
   </Segment>
 ) : (
-  <Segment style={{ borderColor: theme.colours.maroon }}>
+  <Segment style={SegmentStyle} s>
```
[the remark this answers is not in the document]
Comment (OoiYueYing) [reply src/components/DetailCard/DetailCard.jsx:32]: Spot on! it's a typo HAHA dont know why it didnt sound out as an error in VS code

[the remark this answers is not in the document]
Comment (ooihuiying) [reply src/components/DetailCard/DetailCard.jsx]: loll

File: src/components/DetailCard/DetailCard.jsx (hunk 1 above)
Comment (OoiYueYing) [reply src/components/DetailCard/DetailCard.jsx]: good idea!

**Pull request merged**